dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -158,13 +159,32 @@ services:
     restart: always
 
 ########################
+# AGENTS
+########################
+  agents:
+    image: <%= ENV["IMAGE_AGENTS"] %>
+    volumes:
+      - '<%= scope.lookupvar('compose::codenvy_folder') -%>/config/lighttpd/lighttpd.conf:/etc/lighttpd/lighttpd.conf'
+<% if scope.lookupvar('compose::env') != 'production' -%>
+    ports:
+      - '9000:9000'
```

Comment:
either code is removed or either the port is checked in dev mode (as we do for debug port)